_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -1,17 +1,212 @@
+
+| Attribute Type | Attribute Owner | Third Party | Possible? | Automation                                                   | Examples/Reason                                                                                                                                                                                                                                                                                                   |
+| -------------- | --------------- | ----------- | :-------: | ------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity       | Sender          |             |     ✗     | `N/A`                                                        | It makes no sense to read own IdentityAttributes.                                                                                                                                                                                                                                                                 |
+| Identity       | Recipient       |             |     ✓     | `USER_DECISION`                                              | Company asks customer for its delivery address.                                                                                                                                                                                                                                                                   |
+| Relationship   | Sender          |             |     ✓     | `USER_DECISION`                                              | With this combination the **Sender gives the Recipient the one-time permission** to write a RelationshipAttribute once.<br>Example: Electricity provider asks new customers for electricity meter number.                                                                                                         |
+| Relationship   | Recipient       |             |     ✓     | `USER_DECISION`                                              | With this combination the **Sender asks the Recipient for the one-time permission** to write a RelationshipAttribute.<br>Example: Company asks new customer to subscribe to the newsletter and proposes the subscription as default once.                                                                         |
```

Comment:
the user has the option of entering the value of the attribute when this is created. this does not fit directly with the example. An example that differs from the realtionship attribute of the sender is difficult, as (to my knowledge) no difference is visible in the app. 
perhaps the meter reading of the electricity meter is a plausible example

Reply:
That's right, the "proposes the subscription as default once" part fits better for the ProposeAttributeRequestItem. The meter reading of the electricity meter example is good, but is unfortunately already used for the RelationshipAttribute with `owner=<Sender>`. I have now changed the example here so that the company asks the customer for a newsletter subscription, but does not propose a default value.